Pouze v HTTP-Server-Simple-0.38p: blib
Pouze v HTTP-Server-Simple-0.38p: Makefile
Pouze v HTTP-Server-Simple-0.38p: Makefile.old
Pouze v HTTP-Server-Simple-0.38p: pm_to_blib
diff -u -r HTTP-Server-Simple-0.38/t/01live.t HTTP-Server-Simple-0.38p/t/01live.t
--- HTTP-Server-Simple-0.38/t/01live.t	2009-01-02 17:17:28.000000000 +0100
+++ HTTP-Server-Simple-0.38p/t/01live.t	2009-07-27 23:51:06.215623600 +0200
@@ -7,7 +7,7 @@
 # This script assumes that `localhost' will resolve to a local IP
 # address that may be bound to,
 
-my $PORT = 8000 + $$;
+my $PORT = 40000 + int(rand(10000));
 
 
 use HTTP::Server::Simple;
@@ -40,7 +40,7 @@
     my $s=HTTP::Server::Simple::CGI->new($PORT);
     $s->host("localhost");
     my $pid=$s->background();
-    diag("started server on $pid");
+    diag("started server PID=$pid");
     like($pid, '/^-?\d+$/', 'pid is numeric');
     select(undef,undef,undef,0.2); # wait a sec
     my $content=fetch("GET / HTTP/1.1", "");
diff -u -r HTTP-Server-Simple-0.38/t/04cgi.t HTTP-Server-Simple-0.38p/t/04cgi.t
--- HTTP-Server-Simple-0.38/t/04cgi.t	2009-01-02 17:13:08.000000000 +0100
+++ HTTP-Server-Simple-0.38p/t/04cgi.t	2009-07-27 23:46:23.560527400 +0200
@@ -3,7 +3,7 @@
 use strict;
 
 plan tests => 22;
-my $PORT = 8000 + $$;
+my $PORT = 40000 + int(rand(10000));
 
 my $host = gethostbyaddr(inet_aton('localhost'), AF_INET);
 
@@ -60,13 +60,13 @@
   }
 
   like(
-       fetch("GET /cgitest/REQUEST_URI?foo%3Fbar",""),
+       fetch("GET /cgitest/REQUEST_URI?foo%3Fbar"),
        qr/foo%3Fbar/,
        "Didn't decode already"
       );
 
   like(
-       fetch("GET /cgitest/foo%2Fbar/PATH_INFO",""),
+       fetch("GET /cgitest/foo%2Fbar/PATH_INFO"),
        qr|foo/bar|,
        "Did decode already"
       );
